compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -283,10 +283,10 @@
             { ParserRuleContext.CLOSE_PARENTHESIS, ParserRuleContext.REQUIRED_PARAM };
 
     private static final ParserRuleContext[] PARAMETER_START =
-            { ParserRuleContext.ANNOTATIONS, ParserRuleContext.PUBLIC_KEYWORD, ParserRuleContext.TYPE_DESC_IN_PARAM };
+            { ParserRuleContext.ANNOTATIONS, ParserRuleContext.TYPE_DESC_IN_PARAM };
 
     private static final ParserRuleContext[] PARAMETER_WITHOUT_ANNOTS =
-            { ParserRuleContext.PUBLIC_KEYWORD, ParserRuleContext.TYPE_DESC_IN_PARAM };
+            { ParserRuleContext.TYPE_DESC_IN_PARAM };
```

Comment:
With this change we don't have alternative paths here ryt. Can remove `PARAMETER_WITHOUT_ANNOTS` as an alternative path and give `TYPE_DESC_IN_PARAM` as the next rule directly.

Reply:
+1 will do that

Reply:
Fixdd in 785c34a